fake_switches/juniper/juniper_netconf_datastore.py

```diff
@@ -192,6 +192,8 @@ def interface_to_etree(self, port):
                     }
                 }})
 
+        self.apply_trunk_native_vlan_patch(interface_data, port)
```

Comment:
The problem is that in the juniper base, the affectation is dependent of the __ethernet_switiching__ array. In the qfx, the parameter is not set the same way. This is the best way I found to make things work without copying the entire method.